docs/releases/v2020.1.1.rst

```diff
@@ -0,0 +1,17 @@
+Gluon 2020.1.1
+==============
+
+This is the first service release for the Gluon 2020.1.x line, fixing bugs reported by the community.
+
+Bugfixes
+--------
+
+- Fixed non-working LEDs on TP-Link Archer C5 v1 and Archer C7 v2 after an upgrade to Gluon 2020.1.
+
+- Fixed an issue which leads to AVM FRITZ!WLAN Repeater 450E devices being stuck in failsafe mode
+  ater an upgrade to Gluon 2020.1.
+
+Other changes
+-------------
+
+- Linux kernel has been updated to 4.14.171
```

Comment:
Missing known issues.

Reply:
Well, we missed these with the v2020.1 also. But I'll add them back here

Reply:
Also add the regression we know about now.

Reply:
I've included the ER-X one, are there more?

Reply:
That depends on if we want to add the regressions fixed in this release.

Reply:
if we missed them in v2020.1 we should add them to the old release notes now too now that we noticed it

Reply:
Fixed in c219d3c1dc6e5612273daf3f87437df526c400a3 and 9e4eb182908320cb94974663c7d201ce792eea78.